azure-pipelines.yml

```diff
@@ -421,18 +384,30 @@ jobs:
     condition: eq(variables.bitstreamStrategy, 'cached')
     displayName: Prepare cached bitstream
   - bash: |
-      set -e
+      set -ex
+      . util/build_consts.sh
       module load "xilinx/vivado/$(VIVADO_VERSION)"
-      ci/scripts/build-bitstream-vivado.sh top_earlgrey cw310
+      SUB_PATH=hw/top_earlgrey
+      mkdir -p $OBJ_DIR/$SUB_PATH $BIN_DIR/$SUB_PATH
+      ci/bazelisk.sh build //hw/bitstream/vivado:standard --define bitstream=vivado
+      cp -rLvt $OBJ_DIR/$SUB_PATH/ \
+        $($REPO_TOP/bazelisk.sh outquery-all //hw/bitstream/vivado:fpga_cw310)
+      cp -rLvt $BIN_DIR/$SUB_PATH/ \
+        $($REPO_TOP/bazelisk.sh outquery-all //hw/bitstream/vivado:standard)
+
+      # Rename bitstreams to be compatible with subsequent steps
+      # TODO(#13807): replace this after choosing a naming scheme
```

Comment:
Please speak with @a-will about the bitstream archives we're publishing to GCP.  He has some ideas about using more standard filenames and including a manifest file.

We don't need to do that work in this PR, just be aware.

Reply:
Got it -- #13807 is a pre-existing issue that contains the discussion on this I believe. We can revisit it to follow up on this.